tests/test_collections.py

```diff
@@ -21,8 +21,16 @@ class TestCollectionDiscoverPages:
     """
 
     def providers():
-        """Return all collection providers."""
-        return osf_api.get_providers_list(type='collections')
+        """Return collection providers to be used in Discover page test."""
+        all_providers = osf_api.get_providers_list(type='collections')
+        coll_providers = []
+        # The list of collections in some environments (i.e. Staging2) has gotten very
+        # long, so a way to narrow the list is to set allow_submssions to False in the
+        # admin app and we can then skip those old testing collections.
+        for provider in all_providers:
+            if provider['attributes']['allow_submissions']:
+                coll_providers.append(provider)
+        return coll_providers
```

Comment:
golf: `return [p for all_providers if p['attributes']['allow_submissions']]`

Reply:
Fixed. Thanks!